logbook-servlet/pom.xml

```diff
@@ -109,4 +114,63 @@
             <artifactId>json-path-assert</artifactId>
         </dependency>
     </dependencies>
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-shade-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <id>jakarta</id>
+                        <phase>package</phase>
+                        <goals>
+                            <goal>shade</goal>
+                        </goals>
+                        <configuration>
+                            <shadedArtifactAttached>true</shadedArtifactAttached>
+                            <shadedClassifierName>${shadedClassifierName}</shadedClassifierName>
+                            <artifactSet>
+                                <includes>
+                                    <include>${project.groupId}:${project.artifactId}</include>
+                                </includes>
+                            </artifactSet>
+                            <relocations>
+                                <relocation>
+                                    <pattern>jakarta.servlet</pattern>
+                                    <shadedPattern>javax.servlet</shadedPattern>
+                                </relocation>
+                                <relocation>
+                                    <pattern>org.zalando.logbook.servlet</pattern>
+                                    <shadedPattern>org.zalando.logbook.servlet.javax</shadedPattern>
+                                </relocation>
+                            </relocations>
+                        </configuration>
+                    </execution>
+                </executions>
+            </plugin>
+
+            <!-- Make IntelliJ understand the shaded artifact location -->
+            <plugin>
+                <groupId>org.codehaus.mojo</groupId>
+                <artifactId>build-helper-maven-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <id>compile</id>
+                        <phase>package</phase>
+                        <goals>
+                            <goal>attach-artifact</goal>
+                        </goals>
+                        <configuration>
+                            <artifacts>
+                                <artifact>
+                                    <file>${build.directory}/${build.finalName}-${shadedClassifierName}.jar</file>
```

Comment:
build complains about usage of deprecated `build.*` variables, should we use `project.build.*` here?